object/group.go

```diff
@@ -265,7 +259,7 @@ func GetPaginationGroupUsers(groupId string, offset, limit int, field, value, so
 		session = session.Desc(fmt.Sprintf("user.%s", util.SnakeString(sortField)))
 	}
 
-	err := session.Find(&users)
+	err = session.Find(&users)
```

Comment:
Why change this? Is this related to the goal of this PR? If not, then discard it

Reply:
Because the added code above has defined err, use `:=` cannot compile.